package.json

```
@@ -265,7 +265,8 @@
     "@metamask/nonce-tracker@npm:^5.0.0": "patch:@metamask/nonce-tracker@npm%3A5.0.0#~/.yarn/patches/@metamask-nonce-tracker-npm-5.0.0-d81478218e.patch",
     "@metamask/keyring-controller@npm:^16.0.0": "patch:@metamask/keyring-controller@npm%3A17.1.1#~/.yarn/patches/@metamask-keyring-controller-npm-17.1.1-098cb41930.patch",
     "@metamask/keyring-controller@npm:^17.1.0": "patch:@metamask/keyring-controller@npm%3A17.1.1#~/.yarn/patches/@metamask-keyring-controller-npm-17.1.1-098cb41930.patch",
-    "@trezor/connect-web@npm:^9.1.11": "patch:@trezor/connect-web@npm%3A9.3.0#~/.yarn/patches/@trezor-connect-web-npm-9.3.0-040ab10d9a.patch"
+    "@trezor/connect-web@npm:^9.1.11": "patch:@trezor/connect-web@npm%3A9.3.0#~/.yarn/patches/@trezor-connect-web-npm-9.3.0-040ab10d9a.patch",
+    "@metamask/phishing-controller@^10.0.0": "file:./../core/packages/phishing-controller"
```

To access the unreleased changes in https://github.com/MetaMask/core/pull/4526, we need to use a preview build:

```suggestion
    "@metamask/phishing-controller": "npm:@metamask-previews/phishing-controller@11.0.0-preview-3b4ca958"
```

This relative path will not work in a sandboxed CI environment, which is why we're seeing errors like these:

```console
➤ YN0001: Error: @metamask/phishing-controller@file:./../core/packages/phishing-controller::locator=metamask-crx%40workspace%3A.: 
ENOENT: no such file or directory, lstat '/home/runner/work/metamask-extension/core/packages/phishing-controller'
```
> https://github.com/MetaMask/metamask-extension/actions/runs/10589816051/job/29344541600?pr=25839